README.md

```diff
@@ -62,9 +62,12 @@ const [useCounter, sharedCounter$] = connectObservable(
   )
 )
 ```
-Returns a hook that provides the latest update of the accepted observable, and an enhanced 
-version of the original observable (the one that the hook will use), which is shared 
-and "replayable".
+Returns a hook that provides the latest update of the accepted observable, and the
+underlying enhanced observable, which shares the subscription to all of its subscribers,
+and always emits the latest value when subscribing to it.
+
+The shared subscription is closed as soon as there are no subscribers to that
+hook/observable.
```

Comment:
Yep, this description is a lot more accurate than the previous one. I like it.

The only thing that I would change is the end of the last sentence: instead of `hook/observable` let's just say: "observable".